### PR TITLE
[AIRFLOW-5304] Fix extra links in BigQueryOperator with multiple queries

### PR DESCRIPTION
**Pull request by mik-laj**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-5304
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

- [ ] Here are some details about my PR, including screenshots of any UI changes:

CC: @milton0825 

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

File: airflow/contrib/operators/bigquery_operator.py [hunk 1]
```diff
@@ -45,6 +45,30 @@ def get_link(self, operator, dttm):
...
+
+    def get_link(self, operator, dttm):
+        ti = TaskInstance(task=operator, execution_date=dttm)
+        job_ids = ti.xcom_pull(task_ids=operator.task_id, key='job_id')
+        if not job_ids:
+            return None
+        if len(job_ids) < self.index:
+            return None
+        job_id = job_ids[self.index]
+        return 'https://console.cloud.google.com/bigquery?j={job_id}'.format(job_id=job_id)
```
Comment (milton0825) [inline airflow/contrib/operators/bigquery_operator.py]: nit: Can we extract this string to a template and reuse it in line 44?

**Review by milton0825 — APPROVED** [review]
LGTM apart from a small nit. I am glad people are using the extra link feature 😄

File: airflow/contrib/operators/bigquery_operator.py (hunk 1 above)
Comment (mik-laj) [reply airflow/contrib/operators/bigquery_operator.py]: Updated. :-D

**Comment by mik-laj** [comment]
I would like GCP operators to better support these feature.  I hope that we will be able to find hours in the near future to add support for this. I hope to find hours in the near future to add support for this feature to more operators. I am currently working on one set of operators and It will implement this functionality.

**Pull request merged**